hack/make/test-docker-py

```diff
@@ -45,11 +44,12 @@ source hack/make/.integration-test-helpers
 			exec docker build ${build_opts} -t ${docker_py_image} -f tests/Dockerfile "https://github.com/docker/docker-py.git#${DOCKER_PY_COMMIT}"
```

Comment:
note that (trying to solve the issue) I configured the build to run with `--network=host`, so the docker build is running in the same networking namespace as the "outer" container.

perhaps we 'll have to check DNS configuration as well?